Avoids wrong Makefile symlinks (for example Makefile.arm-linux-gnueabihf) and missing CC to compile tiny executables in ./config (cross-compile).
--- configure	2010-05-07 10:12:26.000000000 +0200
+++ configure	2013-07-29 19:40:09.650716890 +0200
@@ -10,17 +10,7 @@
 # This can be disabled if neccessary.
 #
 
-CCS="xlc_r gcc"
-
-#
-# Look for supported compilers
-#
-for c in $CCS; do
-	if CCPATH=`which $c 2>&1` && [ -z "${CCPATH%%/*}" ]; then
-		CC="$c"
-		break
-	fi
-done
+CC=gcc
 
 #
 # Make a link to a proper Makefile.*
